src/content/docs/browser/single-page-app-monitoring/get-started/spa-v2.mdx

```diff
@@ -0,0 +1,96 @@
+* Simplified interaction determination: Interactions are now defined as a UI event (click/keydown/submit -> route change -> DOM modification), offering a clearer and more efficient capture approach.
+* Focus on key metrics: While reported data remains largely unchanged, the new agent no longer tracks JavaScript execution or callback duration within interactions, streamlining the reported information.
+* Reduced interaction durations: Expect significantly shorter interaction durations, particularly for route changes. Initial page load interactions should remain stable.
+* Potential event disassociation: You might observe previously associated `AjaxRequest` and `JavascriptError` events becoming disassociated from interactions, reflecting the focus on UI-driven interactions.
+* New API: The function [`newrelic.interaction({ waitForEnd: true })`](/docs/browser/new-relic-browser/browser-apis/interaction/) allows you to customize when a user interaction ends.
+* Deprecated API: The function `createTracer`: While still functional, the `createTracer` function is deprecated as it no longer keeps interactions open or times callbacks. Note: `BrowserTiming` events are no longer created when using `createTracer` with the new SPA feature.
+
+## Try out the new browser SPA monitoring experience [#enable-feature]
+
+<Steps>
+
+    <Step>
+        ### Review the requirements
+
+        This feature is tested and supported in line with our standard [browser support statement](https://docs.newrelic.com/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes/#support-statement).
+
+
+    </Step>
+
+    <Step>
+        ### Add the feature flag to your browser agent
+
+        If your agent was installed with the [APM method](https://docs.newrelic.com/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent/#apm-agent), contact our support team and we'll enable the feature flag on your account.
+        
+        If your agent was installed with NPM or the UI-based, [copy/paste method](https://docs.newrelic.com/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent/#copy-paste), add the following feature flag your browser agent code:
+
+            1. Find the New Relic browser agent code in your webpage HTMl.
+
+            2. In the `init` configuration object, add the `soft_nav` feature flag. Here's an example:
+
+                ```js
+                <script type="text/javascript"> ;window.NREUM||(NREUM={});init={ …, feature_flags: ['soft_nav'] }:
+                ```
+
+            3. Deploy your app.
+
+        Need to disable this feature? Simply remove the feature flag.
+            
+    </Step>
+
+    <Step>
+        ### Confirm data is being sent to New Relic
+
+        The first thing to check is asking yourself, “Does my interaction follow the heuristic described above?” Make sure the conditions are met for the default interaction to finish.
```

Comment:
@cwli24  How do they check that their interaction follows the heuristic/what conditions do they need to meet? It says it's described above, but I'm not sure where.

Reply:
Stated here:

> * Simplified interaction determination: Interactions are now defined as a UI event (click/keydown/submit -> route change -> DOM modification), offering a clearer and more efficient capture approach.